BenchmarkDotNet/project.json

```diff
@@ -48,6 +48,27 @@
         "System.Management": "4.0.0.0",
         "System.Reflection": "4.0.0.0"
       }
+    },
+    "dnxcore50": {
+      "compilationOptions": {
+        "define": [ "CORE" ]
+      },
+      "dependencies": {
+        "NETStandard.Library": "1.0.0-rc2-23811",
```

Comment:
These dependencies are a bit of a mismatch of stuff. They should be made consistent.